rubocop.yml

```diff
@@ -62,3 +62,15 @@ Style/MultilineBlockChain:
   
 Style/SymbolArray:
   EnforcedStyle: brackets
+
+Style/WordArray:
+  EnforcedStyle: brackets
+
+Style/GuardClause:
+  Enabled: false
+
+Style/NumericPredicate:
+  EnforcedStyle: comparison
```

Comment:
This is the only one I'm on the fence with between comparison and just disabling it altogether. I don't agree with the  default at all but if someone wants to do like `if a.present && a.positive?` it doesn't really bother me.

Reply:
yup, I'm ambivalent as well